.pre-commit-config.yaml

```diff
@@ -29,15 +29,11 @@ repos:
       - id: check-ast
   - repo: https://github.com/astral-sh/ruff-pre-commit
     # Ruff version (Used for linting)
-    rev: v0.0.291
```

Comment:
Does it come with the new version? I don't see any related config (for example, line length)

Reply:
yes it was introduced in https://github.com/astral-sh/ruff/releases/tag/v0.0.290, but I used the newer version to get some patches.